clean.go

```diff
@@ -29,6 +25,29 @@ func isCDir(path string) bool {
 	return hFound
 }
 
+func isPBDir(fis []os.FileInfo) bool {
+	var pbFound bool
+	for _, fi := range fis {
+		if fi.IsDir() {
+			continue
+		}
+		ext := filepath.Ext(fi.Name())
+		if ext != ".proto" {
```

Comment:
Thanks for an explanation. I think it's okay because otherwise directory will be left because it's imported in go code.